server/api.js

```diff
@@ -156,6 +157,33 @@ router.post("/abilities", authorization, async (req, res) => {
     res.json("inserted");
   }
 });
+//<-------------------deselect learning obj btns---------------------->
+
+router.put("/deselect", async (req, res)=>{
```

Comment:
🔴 notice this doesn't have any `authorisation` middleware, so anyone (don't have to be logged in) can deselect the abilities of any user